shared/clearwater/sticky_component.rb

```diff
@@ -1,79 +1,48 @@
 require 'clearwater/component'
-require 'clearwater/black_box_node'
 
 module Clearwater
   class StickyComponent
     include Clearwater::Component
 
-    attr_accessor :props, :renderer
-    attr_reader :children
-
-    def self.render props=nil, children=nil
-      unless `props === nil || props.$$is_hash`
-        children = props
-        props = nil
+    class << self
+      alias_method :_new, :new
+      def new(*args)
+        Wrapper.new(*args) { |*arguments| _new(*arguments) }
```

Comment:
Yeah, it *should* be able to just use `*args`, but I had issues when I didn't explicitly define `*arguments`. Opal on occasion does some mysterious things with passing blocks, and when I see something weird happening, I default to just being verbose.